extensions/file/README.md

```diff
@@ -32,14 +32,23 @@ Please be aware that the integer values (always unsigned) given for the sizes (e
 
 ### Data Types
 
+The data type gives information about the values in the file.
+This can be used to indicate the (maximum) range of numerical values expected.
+For example `unit8` indicates that the numbers are in a range between 0 and 255, 
```

Comment:
```suggestion
For example `uint8` indicates that the numbers are in a range between 0 and 255, 
```

Reply:
LOL - "unit" is what I type more often it seems. Good catch!